disk.py

```diff
@@ -0,0 +1,57 @@
+import json
+from shutil import copyfile
+from os import listdir
+import os.path
+
+DATADIR = os.path.join(os.path.expanduser("~"), ".junction/")
```

Comment:
Not sure what the Windows behavior is here

Reply:
Yup. Need to try that.